### PR TITLE
update rke2 node drain default value

### LINKED ISSUE #9108: Drain Control Plane Nodes and Drain Worker Nodes should be checked default
Internal Ref: SURE-6438

Default to setting both Drain control Plane Nodes and Drain Worker Nodes to checked.

Might be Ember only - should check Dashboard as well.

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
### Summary
Fixes #9108 

Per internal ref, this requested change is for provisioned clusters - see comment on original issue re: imported cluster UI

### Occurred changes and/or fixed issues
* updated the rke2/k3s upgrade strategy configuration to default to drain nodes 'yes'
* updated DrainOptions component to use i18n
* added unit tests for DrainOptions
* added v-clean-tooltip support to jest config (I was getting `Failed to resolve directive: v-clean-tooltip`)

### CODE REVIEW

**Comment by cnotv** [comment]
@mantis-toboggan-md looks good the setup for Jest. Did exactly the same here 
 :D https://github.com/rancher/dashboard/pull/9264

File: shell/edit/provisioning.cattle.io.cluster/DrainOptions.vue [hunk 1]
```diff
@@ -51,6 +52,8 @@ export default {
     this.update();
   },
 
+  computed: { ...mapGetters({ t: 'i18n/t' }) },
```
[the remark this answers is not in the document]
Comment (cnotv) [reply shell/edit/provisioning.cattle.io.cluster/DrainOptions.vue:55]: Nvm, we are not using the CRUResource mixin here.

**Review by cnotv — APPROVED** [review]
LGTM

**Comment by mantis-toboggan-md** [comment]
The internal ticket was updated with more info: the actual field to change is the 'drain' options in the machine pool advanced section. I've updated this PR to set that true by default, added a unit test, and reverted the upgrade strategy defaults change. 

Screen Shot 2023-09-27 at 2 58 04 PM

**Review by momesgin — APPROVED** [review]
LGTM

**Pull request merged**